spring-boot-actuator-docs/src/main/asciidoc/health.adoc

```diff
@@ -17,4 +17,4 @@ Example HTTP response:
 include::{generated}/health/http-response.adoc[]
 
 Example HTTP response with `endpoints.health.sensitive=false`:
-include::{generated}/health/unsensitive/http-response.adoc[]
+include::{generated}/health/insensitive/http-response.adoc[]
```

Comment:
Like it or not but that one is actually legit. Maybe we should change it though.